### ISSUE TITLE
Suggestion:  Duplicate zone with new repeater

### ISSUE THREAD

**Issue by khaytsus** [DaleFarnsworth/codeplug#13]
I've found myself repeating the same procedure over and over and think it could be done easier.  I have 15 or so TG's I want to be able to use, it would be really convenient to be able to go into an existing Zone I've already set up and duplicate it.  During the duplication it would let you fill in the frequency, or perhaps pick an existing channel to use as a source, maybe auto appends the source channel and TG name as the channel name.  Not sure what specifically to suggest for a UI.  But something that could, ideally, duplicate a whole zone with minimal work.

**Comment by DaleFarnsworth** [comment]
I agree this would be useful.  I don't know if or when I'll get to it though.
